manual/configuring_fluidity.tex

```diff
@@ -2652,7 +2652,26 @@ \subsubsection{Inter-phase momentum interaction}
 \mathbf{F}_D = \left(150\frac{\alpha_p^2\mu_f}{\alpha_f d_p^2} + 1.75\frac{\alpha_p\rho_f|\mathbf{u}_f-\mathbf{u}_p|}{d_p}\right)\left(\mathbf{u}_f-\mathbf{u}_p\right).
 \end{equation}
 
-Note that within each dispersed phase, a value for $d$ must be specified in \option{../multiphase\_properties/particle\_diameter} regardless of which drag correlation is used.
+The interphase drag force $\mathbf{F}_D$, and the drag coefficient $C_D$ for the standard \cite{schiller1935drag} drag correlation is given by:
+\begin{equation}\label{eq:schillernaumann_drag_force}
+\mathbf{F}_D = \frac{3\ \alpha_p\ C_D\ \alpha_f\ \rho_f\ |\mathbf{u}_f-\mathbf{u}_p|\ (\mathbf{u}_f-\mathbf{u}_p)}{4\ d},
+\end{equation}
+\begin{equation}\label{eq:schillernaumann_drag_coefficient}
+C_D=
+\begin{cases}
+  \dfrac{24}{Re} \left( 1+0.15 {Re}^{0.687} \right)& \text{if}\ Re<1000, \\[2ex]
+  0.44                                               & \text{otherwise},
+\end{cases}
+\end{equation}
+with a slightly different definition for the Reynolds number than above:
+\begin{equation}\label{eq:particle_reynolds_number_2}
+\mathrm{Re} = \frac{\rho_f\ d\ |\mathbf{u}_f-\mathbf{u}_p|}{\mu_f}.
+\end{equation}
+The above drag force correlation was developed for spherical and solid dispersed phase but fluid dispersed particles with slight impurities can be modelled using the same. \cite{schiller1935drag} drag has been used for the modelling of dilute bubble columns successfully. 
+
+Note that within each dispersed phase, a value for $d$ must be specified in \option{../multiphase\_properties/particle\_diameter} regardless of which drag correlation is used. A scalar field can also be used instead of a constant dispersed phase diameter by choosing the \option{../multiphase\_properties/particle\_dia\_use\_scalar\_field} option instead. The latter option is useful when the multiphase model is coupled to the population balance equation.
```

Comment:
This sounds like the ideal place to cite that world-leading paper you recently published in Computers & Chemical Engineering!